pkg/controller/nodeipam/ipam/cloud_cidr_allocator_test.go

```diff
@@ -970,6 +971,120 @@ func TestUpdateCIDRAllocation(t *testing.T) {
 			},
 			expectedUpdate: true,
 		},
+		{
+			name: "one additional device network but without multi-networking enabled",
```

Comment:
Consider: "node has multiple interfaces and multi-networking is disabled; should use default interface for IPAM"